app/_src/kubernetes-ingress-controller/get-started/services-and-routes.md

```diff
@@ -11,7 +11,46 @@ A Service inside Kubernetes is a way to abstract an application that is running
 
 The service object in Kong holds the information of the protocol to use to talk to the upstream service and various other protocol specific settings. The Upstream object defines load-balancing and health-checking behavior.
 
-![translating Kubernetes to Kong](/assets/images/products/kubernetes-ingress-controller/k8s-to-kong.png "Translating k8s resources to Kong")
+<!--vale off-->
+{% mermaid %}
+flowchart LR
+    subgraph Pods
```

Comment:
Same issue here as the architecture image: need an input arrow, should add "request traffic" or "client request" or something like that, as a new object outside all of these subgraphs.